pkg/tsdb/cloudwatch/metric_find_query.go

```diff
@@ -213,6 +214,7 @@ func init() {
 		"Rekognition":           {},
 		"WAF":                   {"Region", "Rule", "RuleGroup", "WebACL"},
 		"AWS/Cassandra":         {"Keyspace", "Operation", "TableName"},
+		"AWS/WAFV2":             {"Region", "Rule", "WebACL"},
```

Comment:
Looks to me as if [RuleGroup](https://docs.aws.amazon.com/waf/latest/developerguide/monitoring-cloudwatch.html#waf-metrics) is among the dimensions:
```suggestion
		"AWS/WAFV2":             {"Region", "Rule", "WebACL", "RuleGroup"},
```